schemas/file.yml

```diff
@@ -225,3 +225,17 @@
 
       short: A fork is additional data associated with a filesystem object.
       example: Zone.Identifer
+
+    - name: origin_referrer_url
+      level: extended
+      type: keyword
+      ignore_above: 8192
+      description: The url of the webpage that linked to the file.
```

Comment:
Capitalize URL here and in the below description

Reply:
Thanks for the suggestion! I have changed url to URL!

Reply:
Thanks for the suggestion! I have changed url to URL!